apis/facebook/api/src/objects/fb_user.e

```diff
@@ -0,0 +1,223 @@
+note
+	description: "Object representing a Facebook User."
+	date: "$Date$"
+	revision: "$Revision$"
+	EIS: "name=Facebook User", "src=https://developers.facebook.com/docs/graph-api/reference/user/", "protocol=uri"
+
+class
+	FB_USER
+
+feature -- Access: Basic Properties
```

Comment:
maybe a make_with_id or make_with_name  would be good, to have either the id or the name as attached attribute.